### PR TITLE
add more log markers for load balancing and kafka connection to contr…

### PR DESCRIPTION
**Pull request by jeremiaswerner**
…oller

Result looks as follows:
```
controller           +0s   [2016-09-23T15:02:05.828Z] [INFO] [#tid_40] [ActionsApi] [marker:controller_loadbalancer_start:44]
controller           +0s   [2016-09-23T15:02:05.829Z] [INFO] [#tid_40] [LoadBalancerService] [marker:controller_kafka_start:45]
controller           +0s   [2016-09-23T15:02:05.863Z] [INFO] [#tid_40] [LoadBalancerService] [marker:controller_kafka_finish:79:34]
controller           +0s   [2016-09-23T15:02:05.864Z] [INFO] [#tid_40] [ActionsApi] [marker:controller_loadbalancer_finish:80:36]

```

### CODE REVIEW

File: core/controller/src/main/scala/whisk/core/controller/Actions.scala [hunk 1]
```diff
@@ -455,13 +455,15 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
             (message.activationId, None)
         }
 
+        val start = transid.started(this, LoggingMarkers.CONTROLLER_LOADBALANCER)
         info(this, s"[POST] action activation id: ${message.activationId}")
```
Comment (rabbah) [inline core/controller/src/main/scala/whisk/core/controller/Actions.scala]: these two should be merged.

File: core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerToKafka.scala [hunk 2]
```diff
@@ -52,13 +56,15 @@ trait LoadBalancerToKafka extends Logging {
...
         getInvoker(msg) map {
+            val start = transid.started(this, LoggingMarkers.CONTROLLER_KAFKA)
             invokerIndex =>
                 val topic = Message.invoker(invokerIndex)
                 val subject = msg.subject()
                 info(this, s"posting topic '$topic' with activation id '${msg.activationId}'")
                 producer.send(topic, msg) map { status =>
                     val counter = updateActivationCount(subject, invokerIndex)
                     info(this, s"user has ${counter} activations posted. Posted to ${status.topic()}[${status.partition()}][${status.offset()}]")
+                    transid.finished(this, start)
```
Comment (rabbah) [inline core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerToKafka.scala]: and the info above this with the finished marker should also be merged.

**Comment by jeremiaswerner** [comment]
@rabbah when collapsing the two log statements to one, could we still explain where the following delay comes from:

```
controller           +0s   [2016-09-22T08:26:01.046Z] [INFO] [#tid_229806] [ActionsApi] [POST] action activation will block on result up to 65000 milliseconds
controller           +5s   [2016-09-22T08:26:06.028Z] [INFO] [#tid_229806] [KafkaProducerConnector] sent message: invoke24[0][232]
controller           +0s   [2016-09-22T08:26:06.028Z] [INFO] [#tid_229806] [LoadBalancerService] user has 916 activations posted. Posted to invoke24[0][232]
```

I'll merge the info log with the marker.

**Comment by rabbah** [comment]
Yes - you'll still get the marker you want (along with the log message).

**Comment by rabbah** [comment]
Removed `ready` label - we use this when the review is complete, and all system checks are completed (and it's usually assigned by the reviewer ;) ).

File: core/controller/src/main/scala/whisk/core/controller/Actions.scala (hunk 1 above)
Comment (jeremiaswerner) [inline core/controller/src/main/scala/whisk/core/controller/Actions.scala]: done

File: core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerToKafka.scala (hunk 2 above)
Comment (jeremiaswerner) [inline core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerToKafka.scala]: done

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/9e0e1dd684d2010ac8bba8bf2ae1648326c8deb1.

**Pull request closed without merging**